doc/CMakeLists.txt

```diff
@@ -58,3 +62,28 @@ if(DOXYGEN)
         COMMENT "Generating documentation with Doxygen"
         VERBATIM)
 endif()
+
+if(REQUIREMENTS)
+    find_program(PANDOC pandoc)
+
+    if(NOT PANDOC)
+        message(SEND_ERROR
+                "'pandoc' executable has not been found."
+                " Please install it to be able to generate the html version of"
+                " the 'requirements' documentation.")
+    endif()
+
+    configure_file(${CMAKE_CURRENT_SOURCE_DIR}/requirements/requirements.css
+        ${CMAKE_CURRENT_BINARY_DIR}/requirements/requirements.css
+        COPYONLY)
+    add_custom_target(requirements-doc
```

Comment:
try with add_custom_command; it should create a make target that has a real output, which avoid re-making it when up-to-date.